package.json

```diff
@@ -18,61 +18,61 @@
-    "@hapi/inert": "^6.0.2",
-    "@hapi/joi": "^17.1.1",
-    "@hapi/scooter": "^6.0.0",
-    "@hapi/vision": "^6.0.1",
-    "aws-sdk": "^2.760.0",
-    "bash": "^0.0.1",
-    "blankie": "^5.0.0",
-    "blipp": "^4.0.1",
-    "clamscan": "^1.3.3",
+    "@hapi/basic": "6.0.0",
+    "@hapi/catbox-redis": "6.0.1",
+    "@hapi/cookie": "11.0.1",
+    "@hapi/crumb": "8.0.0",
+    "@hapi/glue": "8.0.0",
+    "@hapi/h2o2": "9.0.1",
+    "@hapi/hapi": "20.0.3",
+    "@hapi/hoek": "9.1.0",
+    "@hapi/inert": "6.0.2",
+    "@hapi/joi": "17.1.1",
+    "@hapi/scooter": "6.0.0",
+    "@hapi/vision": "6.0.1",
+    "aws-sdk": "2.798.0",
+    "bash": "0.0.1",
+    "blankie": "5.0.0",
+    "blipp": "4.0.2",
+    "clamscan": "1.3.3",
     "defra-logging-facade": "git+https://github.com/DEFRA/defra-logging-facade.git#02459ac",
-    "disinfect": "^1.1.0",
-    "dot-prop": "^5.3.0",
-    "dotenv": "^8.2.0",
-    "figlet": "^1.5.0",
-    "govuk-frontend": "^3.9.1",
-    "hapi-alive": "^2.0.4",
-    "hapi-robots": "^3.2.0",
-    "hapi-router": "^4.0.1",
-    "moment": "^2.29.0",
-    "nunjucks": "^3.2.2",
-    "proxy-agent": "^3.1.1",
-    "request": "^2.88.2",
-    "request-etag": "^2.0.3",
-    "uuid": "^8.3.0"
+    "disinfect": "1.1.0",
+    "dot-prop": "6.0.1",
+    "dotenv": "8.2.0",
+    "figlet": "1.5.0",
+    "govuk-frontend": "3.10.0",
+    "hapi-alive": "2.0.4",
+    "hapi-robots": "3.2.0",
+    "hapi-router": "4.0.1",
+    "moment": "2.29.1",
+    "nunjucks": "3.2.2",
+    "proxy-agent": "4.0.0",
+    "request": "2.88.2",
+    "request-etag": "2.0.3",
+    "uuid": "8.3.1"
   },
   "devDependencies": {
-    "@hapi/lab": "^24.0.0",
-    "browser-sync": "^2.26.12",
-    "code": "^5.2.4",
-    "del": "^5.1.0",
-    "eslint": "^7.9.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "form-data": "^3.0.0",
-    "gulp": "^4.0.2",
-    "gulp-merge": "^0.1.1",
-    "gulp-minify": "^3.1.0",
-    "gulp-sass": "^4.1.0",
-    "node-sass": "^4.14.1",
-    "gulp-sourcemaps": "^2.6.5",
-    "merge-stream": "^2.0.0",
-    "run-sequence": "^2.2.1",
-    "standard": "^14.3.4",
-    "standard-version": "^9.0.0",
-    "stream-to-promise": "^3.0.0"
+    "@hapi/lab": "24.1.0",
+    "browser-sync": "2.26.13",
+    "code": "5.2.4",
+    "del": "6.0.0",
+    "eslint": "7.14.0",
+    "eslint-config-standard": "16.0.2",
+    "eslint-plugin-import": "2.22.0",
+    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-promise": "4.2.1",
+    "eslint-plugin-standard": "5.0.0",
+    "form-data": "3.0.0",
+    "gulp": "4.0.2",
+    "gulp-merge": "0.1.1",
+    "gulp-minify": "3.1.0",
+    "gulp-sass": "4.1.0",
+    "gulp-sourcemaps": "3.0.0",
+    "merge-stream": "2.0.0",
+    "node-sass": "5.0.0",
+    "run-sequence": "2.2.1",
+    "standard": "16.0.3",
+    "standard-version": "9.0.0",
+    "stream-to-promise": "3.0.0"
```

Comment:
You've removed a lot of 'compatible with' indicators, which ties us to exact versions. I'm not sure why you've done this - I would've thought using the ^ signifier, so we get minor updates, or at worse the ~ so we get patches would be more advisable?

Also, I'm not sure but it might make dependabot struggle to keep us up-to-date.

Reply:
I can change it back. I wasn't sure about the convention. In https://github.com/DEFRA/rod-catch-returns-frontend-tests/blob/master/package.json its exact versions, which is why I used exact. But I can see that https://github.com/DEFRA/rod-licensing/blob/develop/package.json uses `^`. I can change it to that.

Reply:
Yeah, leave it with the 'compatible with' indicators. There isn't a specific convention, but that's what was there before and I think is worth keeping.